demo-app/src/main/java/io/opentelemetry/android/demo/ui/shop/products/ProductDetails.kt

```diff
@@ -0,0 +1,73 @@
+package io.opentelemetry.android.demo.ui.shop.products
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.opentelemetry.android.demo.clients.ImageLoader
+import io.opentelemetry.android.demo.gothamFont
+import io.opentelemetry.android.demo.model.Product
+
+@Composable
+fun ProductDetails(product:Product){
+    val imageLoader = ImageLoader(LocalContext.current)
+    val sourceProductImage = imageLoader.load(product.picture)
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Image(
+            bitmap = sourceProductImage.asImageBitmap(),
+            contentDescription = product.name,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(300.dp)
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            text = product.name,
+            fontFamily = gothamFont,
+            fontSize = 24.sp,
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Center
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            text = product.description,
+            color = Color.Gray,
+            textAlign = TextAlign.Justify,
+            fontFamily = gothamFont,
+            fontSize = 16.sp,
+            modifier = Modifier.fillMaxWidth()
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            text = "$${product.priceValue()}",
+            fontFamily = gothamFont,
+            fontSize = 24.sp,
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Center
+        )
+        Spacer(modifier = Modifier.height(32.dp))
```

Comment:
TODO: in a subsequent PR, the user should be able to specify a quantity to add, typically with some kind of integer spin box.